parkings/models/parking.py

```diff
@@ -152,7 +144,7 @@ def save(self, update_fields=None, *args, **kwargs):
             self.normalized_reg_num = (
                 self.normalize_reg_num(self.registration_number))
 
-        super(Parking, self).save(update_fields=update_fields, *args, **kwargs)
+        super(AbstractParking, self).save(update_fields=update_fields, *args, **kwargs)
```

Comment:
You can use the cleaner `super()` without arguments here, since this is a Python 3 code base.